python/docs/source/conf.py

```diff
@@ -194,7 +194,11 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 html_theme_options = {
-    "navbar_end": ["version-switcher"]
+    "navbar_end": ["version-switcher", "theme-switcher"],
+    "logo": {
+        "image_light": "_static/spark-logo-light.png",
+        "image_dark": "_static/spark-logo-dark.png",
```

Comment:
FYI: The default mode for light/dark is `auto`, which will choose a theme based on the system settings from user, but we can choose one of `dark` or `light` as default manually.